cli/test/smokehouse/config/config.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * List of smoke tests excluded per runner. eg: 'cli': ['a11y', 'dbw']
+ * @type {Record<string, Array<string>>}
```

Comment:
maybe

```suggestion
 * @type {Record<'cli'|'bundle'|'devtools', Array<string>>}
```
just to help out future editors of this file? We could do something with the test names too, but almost definitely not worth the effort